examples/custom-service-account/README.md

```diff
@@ -1,6 +1,8 @@
 # Create Service Account With Custom Settings
 
-This example shows how to create a service account to provide Lacework read-only access to a Google Cloud Organization and customizing the account with a few inputs.
+This example shows how to create a service account to provide 
+Lacework read-only access to a Google Cloud project, customizing 
+the account with a few inputs.
```

Comment:
Does this new module actually provide read-only access? or it only creates an empty service account? 🤔

Reply:
good catch. It creates an empty service account with no roles/permissions. I'll update this.